IA-1972 runiasodev sh script only launch dev server

mathvdh opened the pull request:
The script `runiasodev.sh` is not very useful as it is. It can only launch the django devserver not any other django management commands like the tests or migrations.

This is a modification that allows to give additional arguments to the `runiasodev.sh` like `runiasodev.sh mirgrate` and it will effectively run `./manage.py migrate`

Related JIRA tickets : IA-1972

## Self proofreading checklist
- [x] Is my code clear enough and well documented

## How to test

You can launch `./runiasodev.sh showmigrations` for example and it will work.

olethanh reviewed (changes requested):
ok with the general principle but I would rather we don't have a link called here. see the W3 page for more guidance on this https://www.w3.org/QA/Tips/noClickHere but you could for example have : Validate your XLS form before submitting using this **online tool**. Where the bold part is the link

olethanh commented:
rebased on top of master and changed the text

fr
![image](https://user-images.githubusercontent.com/82500/137096268-80828c60-0e5f-4287-b9c9-4a107a7503da.png)

en
![image](https://user-images.githubusercontent.com/82500/137096391-7d2105a2-3e45-4ce5-8c29-a779930dba12.png)

olethanh committed:
Azure Devops ci: Install dev requirement because we were missing black in pipeline execution

olethanh committed:
IA-1973 Vaccine dashboard crash: optimize number of db query

olethanh committed:
IA-1973 Optimize forma endpoint

Do not fetch the whoe scope for each org unit, only the first time we handle this campaign

olethanh committed:
IA-1973 Fix Botswana import

it was failing because there had no Region

olethanh committed:
IA-1973 Do not fetch data from ONA server

for finished campaigns

olethanh committed:
prevent docker from creating docs/ dir, with wrong perms

olethanh committed:
Fix cache delay

olethanh committed:
fix awsebcli following github action error

olethanh committed:
lower version for awsebcli

olethanh committed:
Cleanup/Rebase after starting from wrong branch

mathvdh committed:
IA-1905 : Created the task
replace temporarely the code from the management command
by a call to the task

mathvdh committed:
Moved to a view/task

mathvdh committed:
Merge branch 'main' of github.com:BLSQ/iaso into IA-1905-preparedness-google-sheets-alerts-send-4-emails

mathvdh committed:
Its working but not sure about the args needed

mathvdh committed:
After discussion with Olivier

mathvdh committed:
Done

mathvdh committed:
Cleanup started from wrong branch

mathvdh committed:
Cleanup

mathvdh committed:
Cleanup comments

mathvdh committed:
Cleanup comments

mathvdh committed:
Merge branch 'IA-1972-runiasodev-sh-script-only-launch-dev-server' of github.com:BLSQ/iaso into IA-1972-runiasodev-sh-script-only-launch-dev-server
